domain/plugins/gui/templates/template.go

```diff
@@ -2,16 +2,52 @@ package templates
 
 import (
 	"context"
+	"html/template"
+	"net/http"
+	"path"
+
 	"github.com/d-velop/dvelop-sdk-go/log"
 	"github.com/shurcooL/httpfs/html/vfstemplate"
-	"html/template"
 )
 
 func parseTemplates() *template.Template {
-	t, e := vfstemplate.ParseGlob(TemplateFileSystem, nil, "*")
+	files, err := listFiles(TemplateFileSystem)
+	if err != nil {
+		panic(err)
```

Comment:
I think you should log the error before panic.

Reply:
Done